backend/src/src-licenses/pom.xml

```diff
@@ -31,7 +31,14 @@
     <dependencies>
         <dependency>
             <groupId>org.spdx</groupId>
-            <artifactId>spdx-tools</artifactId>
+            <artifactId>tools-java</artifactId>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.googlecode.json-simple/json-simple -->
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+            <version>1.1.1</version>
```

Comment:
Kindly define the version in parent (main) `pom.xml` file, and include it here.
also consider removing the maven repo link in comment on line `36`.